templates/includes/more-records.html

```diff
@@ -0,0 +1,23 @@
+{% load static %}
+{% load wagtailimages_tags %}
+{% load wagtailcore_tags %}
+
+<div class="more-records">
```

Comment:
These related pages do not exist yet, I can leave the template but cannot check it as the BE it was made for no longer exists.

Reply:
👍